web/templates/settings/bot_avatar_row.hbs

```diff
@@ -8,8 +8,10 @@
                 <button type="submit" class="btn open_edit_bot_form tippy-zulip-delayed-tooltip" data-sidebar-form="edit-bot" data-tippy-content="{{t 'Edit bot' }}" data-email="{{email}}">
                     <i class="fa fa-pencil blue" aria-hidden="true"></i>
                 </button>
-                <a type="submit" download="{{zuliprc}}" class="btn download_bot_zuliprc tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Download zuliprc' }}" data-email="{{email}}">
-                    <i class="fa fa-download sea-green" aria-hidden="true"></i>
+                <a type="submit" download="{{zuliprc}}"  class="btn download_bot_zuliprc tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Download zuliprc' }}" data-email="{{email}}">
```

Comment:
Since we are adding a button tag inside the anchor tag there is no need to have this `btn` class.

This also leads to a scroll bar being added to the box as well as irregular spacing between the icons.

<details>

<summary> See scrollbar and irregular spacing</summary>

![image](https://github.com/zulip/zulip/assets/35286603/d7dec6b5-8c49-4bef-87c8-8ee1101266d9)

</details>